### ISSUE TITLE
  ld: framework not found Flutter

### ISSUE THREAD

**Issue by oguzhanozdemir** [flutter/flutter#73466]
I have a problem, but I don't find any solution about this. 
Last week I send 4 new update to AppleStore in this app. But suddenly it's broken.

<details>
<summary>logs</summary>

```bash
Launching lib/main.dart on iPhone 12 Pro in debug mode...
Running pod install...
CocoaPods' output:
↳
      Preparing

    Analyzing dependencies

    Inspecting targets to integrate
      Using `ARCHS` setting to build architectures of target `Pods-Runner`: (``)

    Fetching external sources
    -> Fetching podspec for `Flutter` from `Flutter`
    -> Fetching podspec for `device_info` from `.symlinks/plugins/device_info/ios`
    -> Fetching podspec for `onesignal_flutter` from `.symlinks/plugins/onesignal_flutter/ios`
    -> Fetching podspec for `package_info` from `.symlinks/plugins/package_info/ios`
    -> Fetching podspec for `path_provider` from `.symlinks/plugins/path_provider/ios`
    -> Fetching podspec for `shared_preferences` from `.symlinks/plugins/shared_preferences/ios`
    -> Fetching podspec for `syncfusion_flutter_pdfviewer` from `.symlinks/plugins/syncfusion_flutter_pdfviewer/ios`
    -> Fetching podspec for `url_launcher` from `.symlinks/plugins/url_launcher/ios`

    Resolving dependencies of `Podfile`
      CDN: trunk Relative path: CocoaPods-version.yml exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_c_1_2.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/c/1/2/OneSignal/3.1.0/OneSignal.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/c/1/2/OneSignal/2.16.1/OneSignal.podspec.json exists! Returning local because checking is only perfomed in repo update

    Comparing resolved specification to the sandbox manifest
      A Flutter
      A OneSignal
      A device_info
      A onesignal_flutter
      A package_info
      A path_provider
      A shared_preferences
      A syncfusion_flutter_pdfviewer
      A url_launcher

    Downloading dependencies

    -> Installing Flutter (1.0.0)

    -> Installing OneSignal (2.16.1)
     > Git download
     > Git download
         $ /usr/bin/git clone https://github.com/OneSignal/OneSignal-iOS-SDK.git /var/folders/rh/qtgsqwqx5w17khkgsx6rlc_c0000gn/T/d20210107-38642-1bdqql8 --template= --single-branch --depth 1 --branch 2.16.1
      > Copying OneSignal from `/Users/ogzhnozdmr/Library/Caches/CocoaPods/Pods/Release/OneSignal/2.16.1-5f651` to `Pods/OneSignal`

    -> Installing device_info (0.0.1)

    -> Installing onesignal_flutter (2.6.2)

    -> Installing package_info (0.0.1)

    -> Installing path_provider (0.0.1)

    -> Installing shared_preferences (0.0.1)

    -> Installing syncfusion_flutter_pdfviewer (0.0.1)

    -> Installing url_launcher (0.0.1)
      - Running pre install hooks
    [!] The 'Pods-Runner' target has transitive dependencies that include statically linked binaries: (/Users/ogzhnozdmr/Documents/FlutterProjects/durma_onay_asistani/ios/Flutter/Flutter.framework)

    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.0/lib/cocoapods/installer/xcode/target_validator.rb:84:in `block (2 levels) in verify_no_static_framework_transitive_dependencies'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.0/lib/cocoapods/installer/xcode/target_validator.rb:74:in `each_key'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.0/lib/cocoapods/installer/xcode/target_validator.rb:74:in `block in verify_no_static_framework_transitive_dependencies'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.0/lib/cocoapods/installer/xcode/target_validator.rb:73:in `each'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.0/lib/cocoapods/installer/xcode/target_validator.rb:73:in `verify_no_static_framework_transitive_dependencies'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.0/lib/cocoapods/installer/xcode/target_validator.rb:38:in `validate!'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.0/lib/cocoapods/installer.rb:595:in `validate_targets'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.0/lib/cocoapods/installer.rb:162:in `install!'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.0/lib/cocoapods/command/install.rb:52:in `run'
    /Library/Ruby/Gems/2.6.0/gems/claide-1.0.3/lib/claide/command.rb:334:in `run'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.0/lib/cocoapods/command.rb:52:in `run'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.0/bin/pod:55:in `<top (required)>'
    /usr/local/bin/pod:23:in `load'
    /usr/local/bin/pod:23:in `<main>'

Error output from CocoaPods:
↳
         Cloning into '/var/folders/rh/qtgsqwqx5w17khkgsx6rlc_c0000gn/T/d20210107-38642-1bdqql8'...
         Note: switching to '0ecfbe8f5887945eed8c539bbde74b1e943fb6d4'.

         You are in 'detached HEAD' state. You can look around, make experimental
         changes and commit them, and you can discard any commits you make in this
         state without impacting any branches by switching back to a branch.

         If you want to create a new branch to retain commits you create, you may
         do so (now or later) by using -c with the switch command. Example:

           git switch -c <new-branch-name>

         Or undo this operation with:

           git switch -

         Turn off this advice by setting config variable advice.detachedHead to false


    [!] Automatically assigning platform `iOS` with version `9.0` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

Error running pod install
Error launching application on iPhone 12 Pro.
```
</details>

**Comment by TahaTesser** [comment]
Hi @oguzhanozdemir 
This looks like a Cocoapods issue than Flutter isuee. 
Can you please provide your `flutter doctor -v` your `flutter run --verbose` and a complete reproducible minimal code sample
Thank you

**Comment by oguzhanozdemir** [comment]
Hi @TahaTesser 
my **flutter doctor -v** result

[✓] Flutter (Channel stable, 1.22.5, on Mac OS X 10.15.5 19F101 darwin-x64, locale tr-TR)
    • Flutter version 1.22.5 at /Users/ogzhnozdmr/development/flutter
    • Framework revision 7891006299 (4 weeks ago), 2020-12-10 11:54:40 -0800
    • Engine revision ae90085a84
    • Dart version 2.10.4

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.0)
    • Android SDK at /Users/ogzhnozdmr/Library/Android/sdk
    • Platform android-30, build-tools 30.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.2, Build version 12B45b
    • CocoaPods version 1.10.0

[!] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[!] Connected device
    ! No devices available

! Doctor found issues in 2 categories.


my **flutter run --verbose** result

[ +155 ms] executing: [/Users/ogzhnozdmr/development/flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +74 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] 78910062997c3a836feee883712c241a5fd22983
[        ] executing: [/Users/ogzhnozdmr/development/flutter/] git tag --points-at HEAD
[  +33 ms] Exit code 0 from: git tag --points-at HEAD
[        ] 1.22.5
[  +10 ms] executing: [/Users/ogzhnozdmr/development/flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +22 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[   +1 ms] origin/stable
[        ] executing: [/Users/ogzhnozdmr/development/flutter/] git ls-remote --get-url origin
[  +33 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms] https://github.com/flutter/flutter.git
[  +80 ms] executing: [/Users/ogzhnozdmr/development/flutter/] git rev-parse --abbrev-ref HEAD
[  +15 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[   +9 ms] executing: sw_vers -productName
[  +43 ms] Exit code 0 from: sw_vers -productName
[        ] Mac OS X
[        ] executing: sw_vers -productVersion
[  +19 ms] Exit code 0 from: sw_vers -productVersion
[        ] 10.15.5
[        ] executing: sw_vers -buildVersion
[  +42 ms] Exit code 0 from: sw_vers -buildVersion
[   +1 ms] 19F101
[   +1 ms] executing: sysctl hw.optional.arm64
[   +5 ms] Exit code 1 from: sysctl hw.optional.arm64
[        ] sysctl: unknown oid 'hw.optional.arm64'
[  +72 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +4 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +31 ms] executing: /Users/ogzhnozdmr/Library/Android/sdk/platform-tools/adb devices -l
[  +34 ms] executing: /usr/bin/xcode-select --print-path
[   +7 ms] Exit code 0 from: /usr/bin/xcode-select --print-path
[        ] /Applications/Xcode.app/Contents/Developer
[   +1 ms] executing: /usr/bin/xcodebuild -version
[+1181 ms] Exit code 0 from: /usr/bin/xcodebuild -version
[        ] Xcode 12.2
           Build version 12B45b
[   +4 ms] executing: sysctl hw.optional.arm64
[   +7 ms] Exit code 1 from: sysctl hw.optional.arm64
[        ] sysctl: unknown oid 'hw.optional.arm64'
[        ] executing: xcrun xcdevice list --timeout 2
[  +18 ms] xcrun simctl list --json devices
[   +1 ms] executing: xcrun simctl list --json devices
[  +78 ms] List of devices attached
[ +132 ms] {
                      "devices" : {
                        "com.apple.CoreSimulator.SimRuntime.watchOS-7-1" : [

                        ],
                        "com.apple.CoreSimulator.SimRuntime.tvOS-14-2" : [

                        ],
                        "com.apple.CoreSimulator.SimRuntime.iOS-14-2" : [
                          {
                            "dataPath" : "\/Users\/ogzhnozdmr\/Library\/Developer\/CoreSimulator\/Devices\/3775236E-5B05-4777-B75A-A6AAD7D9FD85\/data",
                            "logPath" : "\/Users\/ogzhnozdmr\/Library\/Logs\/CoreSimulator\/3775236E-5B05-4777-B75A-A6AAD7D9FD85",
                            "udid" : "3775236E-5B05-4777-B75A-A6AAD7D9FD85",
                            "isAvailable" : true,
                            "deviceTypeIdentifier" : "com.apple.CoreSimulator.SimDeviceType.iPhone-12-Pro",
                            "state" : "Shutdown",
                            "name" : "iPhone 12 Pro"
                          }
                        ]
                      }
                    }
[+3596 ms] [
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "14.2 (18B79)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPhone13,3",
                        "identifier" : "3775236E-5B05-4777-B75A-A6AAD7D9FD85",
                        "architecture" : "x86_64",
                        "modelUTI" : "com.apple.iphone-12-pro-1",
                        "modelName" : "iPhone 12 Pro",
                        "name" : "iPhone 12 Pro"
                      }
                    ]
[   +2 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[        ] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[ +119 ms] No supported devices connected.
[  +24 ms] "flutter run" took 5.259ms.
[ +112 ms] 
                    #0      throwToolExit (package:flutter_tools/src/base/common.dart:14:3)
                    #1      RunCommand.validateCommand (package:flutter_tools/src/commands/run.dart:353:7)
                    <asynchronous suspension>
                    #2      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:933:11)
                    <asynchronous suspension>
                    #3      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:836:33)
                    <asynchronous suspension>
                    #4      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart)
                    #5      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:29)
                    #6      _rootRun (dart:async/zone.dart:1190:13)
                    #7      _CustomZone.run (dart:async/zone.dart:1093:19)
                    #8      _runZoned (dart:async/zone.dart:1630:10)
                    #9      runZoned (dart:async/zone.dart:1550:10)
                    #10     AppContext.run (package:flutter_tools/src/base/context.dart:149:18)
                    #11     FlutterCommand.run (package:flutter_tools/src/runner/flutter_command.dart:825:20)
                    #12     CommandRunner.runCommand (package:args/command_runner.dart:197:27)
                    #13     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:335:21)
                    #14     _rootRunUnary (dart:async/zone.dart:1198:47)
                    #15     _CustomZone.runUnary (dart:async/zone.dart:1100:19)
                    #16     _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
                    #17     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
                    #18     Future._propagateToListeners (dart:async/future_impl.dart:725:32)
                    #19     Future._completeWithValue (dart:async/future_impl.dart:529:5)
                    #20     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:40:15)
                    #21     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:311:13)
                    #22     FlutterVersion.checkFlutterVersionFreshness (package:flutter_tools/src/version.dart)
                    #23     _rootRunUnary (dart:async/zone.dart:1198:47)
                    #24     _CustomZone.runUnary (dart:async/zone.dart:1100:19)
                    #25     _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
                    #26     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
                    #27     Future._propagateToListeners (dart:async/future_impl.dart:725:32)
                    #28     Future._completeWithValue (dart:async/future_impl.dart:529:5)
                    #29     Future._asyncCompleteWithValue.<anonymous closure> (dart:async/future_impl.dart:567:7)
                    #30     _rootRun (dart:async/zone.dart:1190:13)
                    #31     _CustomZone.run (dart:async/zone.dart:1093:19)
                    #32     _CustomZone.runGuarded (dart:async/zone.dart:997:7)
                    #33     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1037:23)
                    #34     _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
                    #35     _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
                    #36     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:118:13)
                    #37     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:169:5)
                    
                    
[ +484 ms] ensureAnalyticsSent: 257ms
[   +1 ms] Running shutdown hooks
[        ] Shutdown hooks complete
[        ] exiting with code 1

**Comment by oguzhanozdemir** [comment]
@TahaTesser Can you help me? 
How can I solve this problem?

**Comment by TahaTesser** [comment]
Hi @oguzhanozdemir 
1. Please make sure you're connected to a device when running `flutter run --verbose` and provide complete logs which include the issue logs
2. As mentioned this looks like a cocoapods issue rather than Flutter, please try a new project with one of the packages you're using that might causing the issue if there is no issue even then. Try to isolate which package or code of your project is causing this issue and provide a [minimal complete reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example)?
Thank you

**Comment by oguzhanozdemir** [comment]
Hi @TahaTesser 
I try to run a new project on my mac. But also I have same issues. I think you are correct. I try to remove cocoapods on my mac and re install and reboot, after create new flutter app. When I try, I will be turn back.

Thank you

**Comment by oguzhanozdemir** [comment]
Hi @TahaTesser 

I reinstall cocoapods, create new app and run ios.

```
ld: framework not found Flutter
clang: error: linker command failed with exit code 1 (use -v to see invocation)
note: Using new build system
note: Building targets in parallel
note: Planning build
note: Constructing build description
```

**Comment by TahaTesser** [comment]
Hi @oguzhanozdemir 

Can you please provide  **complete** your `flutter run --verbose > run.log` (please provide run.log file) and a complete reproducible minimal code sample or reproducible steps 
Thank you

**Comment by oguzhanozdemir** [comment]
Hi @TahaTesser ,
I added run.log file.

I didn't develop anything, help me :)

[run.log](https://github.com/flutter/flutter/files/5830524/run.log)

**Comment by TahaTesser** [comment]
Hi @oguzhanozdemir 
I see the buld is succesful 
```bash
               [+1986 ms] debug_universal_framework: Complete
               [   +4 ms] debug_ios_bundle_flutter_assets: Starting due to {}
               [ +521 ms] debug_ios_bundle_flutter_assets: Complete
               [ +892 ms] Persisting file store
               [  +13 ms] Done persisting file store
               [   +8 ms] build succeeded.
               [  +13 ms] "flutter assemble" took 29.552ms.
               [   +5 ms] ensureAnalyticsSent: 0ms
               [   +1 ms] Running shutdown hooks
               [        ] Shutdown hook priority 4
               [        ] Shutdown hooks complete
               [        ] exiting with code 0

```

However, it still throws this error, this is quite different than your original issue https://github.com/flutter/flutter/issues/73466#issue-781115137 
```bash
               ld: framework not found Flutter
               clang: error: linker command failed with exit code 1 (use -v to see invocation)


[   +6 ms] "flutter run" took 68.226ms.
[ +179 ms] ensureAnalyticsSent: 173ms
[   +2 ms] Running shutdown hooks
[        ] Shutdown hook priority 4
[   +4 ms] Shutdown hooks complete
[        ] exiting with code 1
```

Please check out possible solutions [here](https://stackoverflow.com/a/62830644)

**Comment by shivpatil007** [comment]
Hey @TahaTesser  i am facing the same issue 
my  flutter run --verbose is 
[ +236 ms] executing: [C:\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +157 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +2 ms] 78910062997c3a836feee883712c241a5fd22983
[   +2 ms] executing: [C:\flutter/] git tag --points-at HEAD
[ +174 ms] Exit code 0 from: git tag --points-at HEAD
[   +1 ms] 1.22.5
[  +26 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[ +107 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[   +2 ms] origin/stable
[        ] executing: [C:\flutter/] git ls-remote --get-url origin
[  +69 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms] https://github.com/flutter/flutter.git
[ +145 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref HEAD
[  +95 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +1 ms] stable
[ +149 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[  +25 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[  +21 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[  +13 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[  +53 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[  +20 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +51 ms] executing: C:\Users\Shiv\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +96 ms] List of devices attached
           192.168.111.101:5555   device product:vbox86p model:Google_Nexus_5X device:vbox86p transport_id:2
[  +14 ms] C:\Users\Shiv\AppData\Local\Android\sdk\platform-tools\adb.exe -s 192.168.111.101:5555 shell getprop
[ +152 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[  +17 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[  +21 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +4 ms] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[  +22 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[  +17 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +535 ms] Generating G:\flutter project\yoboi\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[  +74 ms] ro.hardware = vbox86
[   +1 ms] ro.build.characteristics = nosdcard
[ +148 ms] Starting incremental build...
[   +6 ms] Initializing file store
[  +45 ms] Skipping target: gen_localizations
[  +20 ms] complete
[  +22 ms] Launching lib\main.dart on Google Nexus 5X in debug mode...
[  +32 ms] C:\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev
C:\flutter\bin\cache\artifacts\engine\windows-x64\frontend_server.dart.snapshot --sdk-root
C:\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --incremental --target=flutter --debugger-module-names
--experimental-emit-debug-metadata -Ddart.developer.causal_async_stacks=true --output-dill
C:\Users\Shiv\AppData\Local\Temp\flutter_tools.403b5417\flutter_tool.146baf07\app.dill --packages .packages -Ddart.vm.profile=false
-Ddart.vm.product=false
--bytecode-options=source-positions,local-var-info,debugger-stops,instance-field-initializers,keep-unreachable-code,avoid-closure-call-instructions --enable-asserts --track-widget-creation --filesystem-scheme org-dartlang-root --initialize-from-dill build\cache.dill.track.dill
[ +109 ms] executing: C:\Users\Shiv\AppData\Local\Android\sdk\platform-tools\adb.exe -s 192.168.111.101:5555 shell -x logcat -v time -t 1
[ +105 ms] Exit code 0 from: C:\Users\Shiv\AppData\Local\Android\sdk\platform-tools\adb.exe -s 192.168.111.101:5555 shell -x logcat -v time -t 1
[   +2 ms] --------- beginning of main
           02-03 06:13:56.018 I/logcat  ( 1563): type=1400 audit(0.0:605): avc: denied { read write } for path="/dev/pts/1" dev="devpts" ino=4    
           scontext=u:r:logd:s0 tcontext=u:object_r:devpts:s0 tclass=chr_file permissive=1
[  +29 ms] executing: C:\Users\Shiv\AppData\Local\Android\sdk\platform-tools\adb.exe -s 192.168.111.101:5555 shell -x logcat -v time -t 1
[ +117 ms] Exit code 0 from: C:\Users\Shiv\AppData\Local\Android\sdk\platform-tools\adb.exe -s 192.168.111.101:5555 shell -x logcat -v time -t 1
[   +1 ms] --------- beginning of main
           02-03 06:13:56.030 I/logd.reader(  116): type=1400 audit(0.0:609): avc: denied { setopt } for path="/dev/socket/logdr"
           scontext=u:r:logd:s0 tcontext=u:r:init:s0 tclass=unix_stream_socket permissive=1
[  +17 ms] <- compile package:yoboi/main.dart
[  +46 ms] executing: C:\Users\Shiv\AppData\Local\Android\sdk\platform-tools\adb.exe version
[ +169 ms] Android Debug Bridge version 1.0.41
                    Version 30.0.5-6877874
                    Installed as C:\Users\Shiv\AppData\Local\Android\sdk\platform-tools\adb.exe
[   +6 ms] executing: C:\Users\Shiv\AppData\Local\Android\sdk\platform-tools\adb.exe start-server
[ +154 ms] Building APK
[  +61 ms] Running Gradle task 'assembleDebug'...
[  +24 ms] gradle.properties already sets `android.enableR8`
[  +42 ms] Using gradle from G:\flutter project\yoboi\android\gradlew.bat.
[  +96 ms] G:\flutter project\yoboi\android\gradlew.bat mode: 33279 rwxrwxrwx.
[  +89 ms] executing: C:\Program Files\Android\Android Studio\jre\bin\java -version
[ +474 ms] Exit code 0 from: C:\Program Files\Android\Android Studio\jre\bin\java -version
[   +3 ms] openjdk version "1.8.0_242-release"
           OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
           OpenJDK 64-Bit Server VM (build 25.242-b01, mixed mode)
[  +35 ms] executing: [G:\flutter project\yoboi\android/] G:\flutter project\yoboi\android\gradlew.bat -Pverbose=true
-Ptarget-platform=android-x86 -Ptarget=G:\flutter project\yoboi\lib\main.dart -Ptrack-widget-creation=true -Pfilesystem-scheme=org-dartlang-root  
assembleDebug
[ +580 ms] Downloading https://services.gradle.org/distributions/gradle-5.6.2-all.zip
[+11942 ms] Exception in thread "main" java.net.UnknownHostException: services.gradle.org
[   +9 ms]      at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
[ +108 ms]      at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
[   +3 ms]      at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
[  +59 ms]      at java.net.Socket.connect(Socket.java:607)
[  +11 ms]      at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:666)
[   +2 ms]      at sun.security.ssl.BaseSSLSocketImpl.connect(BaseSSLSocketImpl.java:173)
[   +7 ms]      at sun.net.NetworkClient.doConnect(NetworkClient.java:180)
[   +2 ms]      at sun.net.www.http.HttpClient.openServer(HttpClient.java:463)
[ +252 ms]      at sun.net.www.http.HttpClient.openServer(HttpClient.java:558)
[  +62 ms]      at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
[  +33 ms]      at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
[  +60 ms]      at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
[  +47 ms]      at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1162)
[   +2 ms]      at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1056)
[   +6 ms]      at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
[   +3 ms]      at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1570)
[ +105 ms]      at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1498)
[   +1 ms]      at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:268)
[   +1 ms]      at org.gradle.wrapper.Download.downloadInternal(Download.java:58)
[   +2 ms]      at org.gradle.wrapper.Download.download(Download.java:44)
[   +2 ms]      at org.gradle.wrapper.Install$1.call(Install.java:61)
[   +2 ms]      at org.gradle.wrapper.Install$1.call(Install.java:48)
[  +34 ms]      at org.gradle.wrapper.ExclusiveFileAccessManager.access(ExclusiveFileAccessManager.java:65)
[  +66 ms]      at org.gradle.wrapper.Install.createDist(Install.java:48)
[   +8 ms]      at org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:128)
[   +1 ms]      at org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:61)
[  +39 ms] Running Gradle task 'assembleDebug'... (completed in 14.2s)
[+3675 ms] Exception: Gradle task assembleDebug failed with exit code 1
[  +11 ms] "flutter run" took 20,202ms.
[  +36 ms] 
           #0      throwToolExit (package:flutter_tools/src/base/common.dart:14:3)
           #1      RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:607:7)
           <asynchronous suspension>
           #2      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:977:18)
           #3      _rootRunUnary (dart:async/zone.dart:1198:47)
           #4      _CustomZone.runUnary (dart:async/zone.dart:1100:19)
           #5      _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
           #6      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
           #7      Future._propagateToListeners (dart:async/future_impl.dart:725:32)
           #8      Future._completeWithValue (dart:async/future_impl.dart:529:5)
           #9      _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:40:15)
           #10     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:311:13)
           #11     RunCommand.usageValues (package:flutter_tools/src/commands/run.dart)
           #12     _rootRunUnary (dart:async/zone.dart:1198:47)
           #13     _CustomZone.runUnary (dart:async/zone.dart:1100:19)
           #14     _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
           #15     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
           #16     Future._propagateToListeners (dart:async/future_impl.dart:725:32)
           #17     Future._completeWithValue (dart:async/future_impl.dart:529:5)
           #18     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:40:15)
           #19     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:311:13)
           #20     AndroidDevice.isLocalEmulator (package:flutter_tools/src/android/android_device.dart)
           #21     _rootRunUnary (dart:async/zone.dart:1198:47)
           #22     _CustomZone.runUnary (dart:async/zone.dart:1100:19)
           #23     _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
           #24     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
           #25     Future._propagateToListeners (dart:async/future_impl.dart:725:32)
           #26     Future._completeWithValue (dart:async/future_impl.dart:529:5)
           #27     Future._asyncCompleteWithValue.<anonymous closure> (dart:async/future_impl.dart:567:7)
           #28     _rootRun (dart:async/zone.dart:1190:13)
           #29     _CustomZone.run (dart:async/zone.dart:1093:19)
           #30     _CustomZone.runGuarded (dart:async/zone.dart:997:7)
           #31     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1037:23)
           #32     _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
           #33     _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
           #34     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:118:13)
           #35     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:169:5)


[ +345 ms] ensureAnalyticsSent: 253ms
[   +2 ms] Running shutdown hooks
[  +16 ms] Shutdown hook priority 4
[  +11 ms] Shutdown hooks complete
[  +22 ms] exiting with code 1



and my flutter run -v is
[ +236 ms] executing: [C:\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +157 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +1 ms] 78910062997c3a836feee883712c241a5fd22983
[   +1 ms] executing: [C:\flutter/] git tag --points-at HEAD
[ +116 ms] Exit code 0 from: git tag --points-at HEAD
[   +1 ms] 1.22.5
[  +18 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[ +174 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[  +14 ms] origin/stable
[   +1 ms] executing: [C:\flutter/] git ls-remote --get-url origin
[ +124 ms] Exit code 0 from: git ls-remote --get-url origin
[   +2 ms] https://github.com/flutter/flutter.git
[ +138 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref HEAD
[  +78 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +1 ms] stable
[ +190 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[  +34 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[  +41 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[  +23 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[  +44 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[  +22 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +4 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +55 ms] executing: C:\Users\Shiv\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +96 ms] List of devices attached
[   +7 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[  +18 ms] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[  +10 ms] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[  +63 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[  +23 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[  +52 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +23 ms] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[ +120 ms] No supported devices connected.
[  +44 ms] "flutter run" took 752ms.
[  +70 ms] 
           #0      throwToolExit (package:flutter_tools/src/base/common.dart:14:3)
           #1      RunCommand.validateCommand (package:flutter_tools/src/commands/run.dart:353:7)
           <asynchronous suspension>
           #2      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:933:11)
           <asynchronous suspension>
           #3      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:836:33)
           <asynchronous suspension>
           #4      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart)
           #5      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:29)
           #6      _rootRun (dart:async/zone.dart:1190:13)
           #7      _CustomZone.run (dart:async/zone.dart:1093:19)
           #8      _runZoned (dart:async/zone.dart:1630:10)
           #9      runZoned (dart:async/zone.dart:1550:10)
           #10     AppContext.run (package:flutter_tools/src/base/context.dart:149:18)
           #11     FlutterCommand.run (package:flutter_tools/src/runner/flutter_command.dart:825:20)
           #12     CommandRunner.runCommand (package:args/command_runner.dart:197:27)
           #13     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:335:21)      
           #14     _rootRunUnary (dart:async/zone.dart:1198:47)
           #15     _CustomZone.runUnary (dart:async/zone.dart:1100:19)
           #16     _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
           #17     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
           #18     Future._propagateToListeners (dart:async/future_impl.dart:725:32)
           #19     Future._completeWithValue (dart:async/future_impl.dart:529:5)
           #20     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:40:15)
           #21     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:311:13)
           #22     FlutterVersion.checkFlutterVersionFreshness (package:flutter_tools/src/version.dart)
           #23     _rootRunUnary (dart:async/zone.dart:1198:47)
           #24     _CustomZone.runUnary (dart:async/zone.dart:1100:19)
           #25     _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
           #26     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
           #27     Future._propagateToListeners (dart:async/future_impl.dart:725:32)
           #28     Future._completeWithValue (dart:async/future_impl.dart:529:5)
           #29     Future._asyncCompleteWithValue.<anonymous closure> (dart:async/future_impl.dart:567:7)
           #30     _rootRun (dart:async/zone.dart:1190:13)
           #31     _CustomZone.run (dart:async/zone.dart:1093:19)
           #32     _CustomZone.runGuarded (dart:async/zone.dart:997:7)
           #33     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1037:23)
           #34     _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
           #35     _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
           #36     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:118:13)
           #37     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:169:5)


[ +711 ms] ensureAnalyticsSent: 258ms
[   +5 ms] Running shutdown hooks
[  +19 ms] Shutdown hooks complete
[  +24 ms] exiting with code 1

**Comment by TahaTesser** [comment]
@shivpatil007 
You're hitting a different issue for Android, this Is a Gradle issue, deleting your Gradle cache and try again with  a device/emulator connected
```
[  +35 ms] executing: [G:\flutter project\yoboi\android/] G:\flutter project\yoboi\android\gradlew.bat -Pverbose=true
-Ptarget-platform=android-x86 -Ptarget=G:\flutter project\yoboi\lib\main.dart -Ptrack-widget-creation=true -Pfilesystem-scheme=org-dartlang-root  
assembleDebug
[ +580 ms] Downloading https://services.gradle.org/distributions/gradle-5.6.2-all.zip
[+11942 ms] Exception in thread "main" java.net.UnknownHostException: services.gradle.org
[   +9 ms]      at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
[ +108 ms]      at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
[   +3 ms]      at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
[  +59 ms]      at java.net.Socket.connect(Socket.java:607)
[  +11 ms]      at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:666)
```

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.

**Comment by Abduboriy3444** [comment]
hi @oguzhanozdemir 
I had the same issue on my MacBook M1 chip.
I solved this quit Android Studio and virtual devices and  by running:
on terminal:
flutter upgrade --force
and in Android Studio project terminal:
arch -x86_64 sudo gem install ffi
rm ios/Podfile

this worked for me
